GuysNight.LethalCompanyMod.BalancedItems/Models/OverrideProperties.cs

```diff
@@ -22,77 +23,39 @@ public sealed class OverrideProperties {
 			Weight = weight;
 		}
 
-		/// <summary>
-		/// The constructor to use when overriding only the item's value is desired.
-		/// </summary>
-		/// <param name="averageValue">The average price that you want the scrap item to be worth.</param>
-		public OverrideProperties(ushort averageValue) {
-			AverageValue = averageValue;
-		}
-
-		/// <summary>
-		/// The constructor to use when overriding the item's weight and value is desired.
-		/// </summary>
-		/// <param name="weight">How much you want the item to weigh.</param>
-		/// <param name="averageValue">The average price that you want the scrap item to be worth.</param>
-		public OverrideProperties(float weight, ushort averageValue) {
-			AverageValue = averageValue;
-			Weight = weight;
-		}
-
 		/// <summary>
 		/// The average value of the item to override the default.
 		/// </summary>
-		private ushort? AverageValue { get; }
+		public ushort AverageValue { get; set; }
 
 		/// <summary>
 		/// The minimum value that the item can sell for.
 		/// </summary>
-		public int? MinValue {
-			get {
-				if (!AverageValue.HasValue) {
-					return null;
-				}
-
-				return (int)Math.Round(AverageValue.Value - AverageValue.Value * .2, MidpointRounding.AwayFromZero);
-			}
-		}
+		public int MinValue => (int)Math.Round(AverageValue - AverageValue * .2, MidpointRounding.AwayFromZero);
```

Comment:
Would it make sense to have the variance defined as a constant ?

Reply:
Probably. I had tossed around the idea of making it configurable too. :) For now, there's no reason to not use a `const` here. However, if we decide to make it configurable, then making it `const` now would be wasted effort.

Reply:
I have made a const for now since it's minimal effort.

Marking resolved.